MAPL_Base/MAPL_HistoryGridComp.F90

```diff
@@ -1409,6 +1414,17 @@ subroutine Initialize ( gc, import, dumexport, clock, rc )
        enddo
     enddo
 
+    ! Deal with split 4d field
+    !--------------------------
+    call split4dFields(rc=status)
+    _VERIFY(status)
+
+    do n=1,nlist
+       m=list(n)%field_set%nfields
+       allocate(list(n)%r4(m), list(n)%r8(m), list(n)%r8_to_r4(m), stat=status)  
+       _VERIFY(STATUS)
+    end do
+    
```

Comment:
Don't change now,  but we really need to start getting in the habit of using Arlindo's __RC__ now that we have collectively agreed on it as the general approach.

Reply:
Sure. Since this is new code, and I expected to find bugs, it was easier to set breakpoints with the "old" style